docs/tre-admins/auth.md

```diff
@@ -9,7 +9,8 @@ The following values are needed to be in place to run the script. (`/templates/c
 | Key | Description |
 | ----------- | ----------- |
 |TRE_ID|This is needed to build up the redirect URI for the Swagger App|
-|AAD_TENANT_ID|The tenant id of where your AAD identities will be placed. This can be different to the tenant where your Azure resources are created.
+|AAD_TENANT_ID|The tenant id of where your AAD identities will be placed. This can be different to the tenant where your Azure resources are created.|
```

Comment:
Can we use the work AUTH in here somewhere to distinguish between the ARM tenant?

Reply:
Just to get this right, we used to have duplicates and have AAD_TENANT_ID and AUTH_TENANT_ID using the same value - now you want AAD_AUTH_TENANT_ID?? This isn't a change I'm making in this PR as it needs many files updating. I think that needs to be a separate story - a valid one I'll grant you!

Reply:
Created #1715